ui/components/multichain/detected-token-banner/detected-token-banner.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import testData from '../../../../.storybook/test-data';
+import configureStore from '../../../store/store';
+import { DetectedTokensBanner } from './detected-token-banner';
+
+const store = configureStore(testData);
+
+export default {
+  title: 'Components/Multichain/DetectedTokensBanner',
+  component: DetectedTokensBanner,
+  decorators: [(story) => <Provider store={store}>{story()}</Provider>],
```

Comment:
Do we need to use the store here if you aren't manipulating the testData being passed? All components are all ready feed the mock state in `main.js`

Reply:
Oh, my bad. 😞